crates/sui-network/src/randomness/mod.rs

```diff
@@ -756,6 +773,40 @@ impl RandomnessEventLoop {
             .expect("RandomnessRoundReceiver mailbox should not overflow or be closed");
     }
 
+    fn maybe_ignore_byzantine_peer(&mut self, epoch: EpochId, peer_id: PeerId) {
+        if epoch != self.epoch {
+            return; // make sure we're still on the same epoch
+        }
+        let Some(dkg_output) = &self.dkg_output else {
+            return; // can't ignore a peer if we haven't finished DKG
+        };
+        if !self.allowed_peers_map.contains(&peer_id) {
+            return; // peer is already disallowed
+        }
+        let Some(peer_share_ids) = &self.peer_share_ids else {
+            return; // can't ignore a peer if we haven't finished DKG
```

Comment:
update comment?

Reply:
the comment is accurate, this can only be None if DKG has not yet finished